brainscore/benchmarks/Lee2020.py

```diff
@@ -0,0 +1,188 @@
+	        elocation-id = {2020.07.09.185116},
+	        year = {2020},
+	        doi = {10.1101/2020.07.09.185116},
+	        publisher = {Cold Spring Harbor Laboratory},
+	        abstract = {A salient characteristic of monkey inferior temporal (IT) cortex is the IT face processing network. Its hallmarks include: {\textquotedblleft}face neurons{\textquotedblright} that respond more to faces than non-face objects, strong spatial clustering of those neurons in foci at each IT anatomical level ({\textquotedblleft}face patches{\textquotedblright}), and the preferential interconnection of those foci. While some deep artificial neural networks (ANNs) are good predictors of IT neuronal responses, including face neurons, they do not explain those face network hallmarks. Here we ask if they might be explained with a simple, metabolically motivated addition to current ANN ventral stream models. Specifically, we designed and successfully trained topographic deep ANNs (TDANNs) to solve real-world visual recognition tasks (as in prior work), but, in addition, we also optimized each network to minimize a proxy for neuronal wiring length within its IT layers. We report that after this dual optimization, the model IT layers of TDANNs reproduce the hallmarks of the IT face network: the presence of face neurons, clusters of face neurons that quantitatively match those found in IT face patches, connectivity between those patches, and the emergence of face viewpoint invariance along the network hierarchy. We find that these phenomena emerge for a range of naturalistic experience, but not for highly unnatural training. Taken together, these results show that the IT face processing network could be a consequence of a basic hierarchical anatomy along the ventral stream, selection pressure on the visual system to accomplish general object categorization, and selection pressure to minimize axonal wiring length.Competing Interest StatementThe authors have declared no competing interest.},
+            URL = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116},
+            eprint = {https://www.biorxiv.org/content/early/2020/07/10/2020.07.09.185116.full.pdf},
+            journal = {bioRxiv}}'''
+
+
+class DicarloLee2020ITSpatialCorrelation(BenchmarkBase):
+
+    def __init__(self):
+        '''
+        This benchmark compares the distribution of pairwise response correlation as a function of distance between the
+            MajajHong assembly and a candidate assembly
+        Plots of this can be found in Figure 2 of the corresponding publication (see BIBTEX)
+        '''
+        assembly = brainscore.get_assembly('dicarlo.MajajHong2015').sel(region='IT')
+        assembly = make_static(assembly)
+
+        super(self).__init__(identifier='dicarlo.Lee2020.IT-tissue',
+                             ceiling_func=lambda: InternalConsistency()(assembly.transpose('presentation', 'neuroid')),

```

Comment:
we'll probably want a different ceiling here that reflects the differences in spatial correlations

Reply:
Effectively there are then two ceilings like we talked about: one for the internal consistency to normalize correlation values, and one for e.g. cross-animal consistency to normalize model scores